build.yml

```diff
@@ -307,6 +307,29 @@ docker-targets:
       xorg-x11-fonts-Type1
       zlib
 
+  lambdaamazonlinux2:
+    source: docker/Dockerfile.centos
+    args:
+      from: amazonlinux:2
+    output: zip
+    postcompile: aws-lambda
+    matrix: ['x86_64']
+    depend: >
```

Comment:
`depend` isn't required, and instead of `matrix` you can just use `arch:   x86_64`